pages/test_analytics/ci_environments.md

```diff
@@ -4,14 +4,25 @@ Buildkite Test Analytics collectors automatically detect common continuous integ
 If available, test collectors gather information about your test runs, such as branch names and build IDs.
 Test collectors gather information from the following CI environments:
 
-- Buildkite
-- CircleCI
-- GitHub Actions
+- [Buildkite](/docs/test-analytics/ci-environments#buildkite)
+- [CircleCI](/docs/test-analytics/ci-environments#circleci)
+- [GitHub Actions](/docs/test-analytics/ci-environments#github-actions)
 
 If you run test collectors inside containers or you're using another CI system, then you must set variables to report your CI details to Buildkite.
 
 If you're not using a test collector, see [Importing JSON](/docs/test-analytics/importing-json) and [Importing JUnit XML](/docs/test-analytics/importing-junit-xml) to learn how to provide run environment data.
 
+## Recommended environment variables
+If you're manually providing Buildkite Test Analytics environment variables, we'd strongly recommend setting the following variables:
+
+| Field name             | Description                                   | Features unlocked                                 |
+|------------------------|-----------------------------------------------|---------------------------------------------------|
+| `run_env[branch]`      | the branch or reference for this build        | filter data by branch                             |
+| `run_env[key]`         | the UUID for the build                        | required field, group batches of data by this key |
```

Comment:
Should required fields go at the top for some data hierarchy 🐘

Reply:
Maybe having a 4th column to indicate optional vs required? I've seen that in other docs, might make us seem less data hungry or something? 🐘

Reply:
> Should required fields go at the top for some data hierarchy 🐘

🤔 I just copied this order from the Buildkite section below, but there's no rhyme or reason as to the order in which these env vars are displayed... The key is actually last in the other sections! I think I will move it to the top in this section but the ordering of this is pretty inconsistent lol 

> Maybe having a 4th column to indicate optional vs required? I've seen that in other docs, might make us seem less data hungry or something? 🐘

Hmm, I don't think the table would look as nice with four columns